docs/extension/guides/create-backup.md

```diff
@@ -8,31 +8,32 @@ description: 'Extension Support: Create Backup'
+- **File Export**: Within the Settings screen, you can export all private keys of the Universal Profile of an extension as text. Make sure to export the text into a file and store it safe. Those private keys will allow anyone to regain access to the profiles based on the controller's permissions.
 
 :::info Individual 2FA Controllers
 
 Please note that each Universal Profile has to add a **separate** [UP Recovery controller](../controllers.md) to [complete the 2FA recovery setup](./check-2fa-recovery.md). If you created multiple Universal Profiles through [UniversalProfile.cloud](https://universalprofile.cloud/), please ensure that you **add the 2FA Recovery** during **every single** deployment process. The connected account and authentication code will stay the same across all profiles of one network and is used to authenticate on the LUKSO Transaction Relay Service.
 
 :::
 
-:::danger Paused Import Functionality
+:::tip
 
-**QR Import** is currently disabled due to the recent [Controller Recovery Incident](../incidents/controller-recovery.md)
+To reduce risk of losing access, we **recommend** setting up **[2FA](../controllers.md#how-can-i-add-2fa-recovery-to-an-existing-universal-profile)** or **[Wallet Recovery](./set-wallet-recovery.md)**.
 
 :::
 
-:::danger Securing Controllers
+:::danger Paused Import Functionality
 
-Due to the nature of [backups and recovery](../controllers.md#what-happens-during-imports-and-recovery), the **controllers of an extension will not be included within 2FA recovery**. If you [fund the extension's controllers](./fund-controller.md) to pay for your transactions, please **always reveal the private key**, e.g., choose **File Export**. Otherwise, you won't be able to [get your controller funds back](./get-controller-funds.md).
+**QR Import** is currently disabled due to the recent [Controller Recovery Incident](../incidents/controller-recovery.md)
 
 :::
 
-:::caution
+:::danger Securing Controllers
 
-The Universal Profile Browser Extension currently does not feature a way to import profiles using the plain private key described in _File Export_. Therefore, we **recommend [setting up 2FA](../controllers.md#how-can-i-add-2fa-recovery-to-an-existing-universal-profile)**. Optionally, the private key of a controller (EOA) could be imported into a regular wallet to send transactions manually. You can find further information in the [Controller](../controllers.md) section.
+Due to the nature of [backups and recovery](../controllers.md#what-happens-during-imports-and-recovery), the **controllers of an extension will not be included within 2FA recovery**. If you [fund the extension's controllers](./fund-controller.md) to pay for your transactions, please **always reveal the private key**, e.g., choose **File Export**. Otherwise, you won't be able to [get your controller funds back](./get-controller-funds.md).
```

Comment:
We can't just write:

> please **always reveal the private key**

This is just odd. "Reveal the private key" means nothing, the real action is "backup the private key".